modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/AddToCartButton.js

```diff
@@ -48,6 +48,9 @@ function AddToCartButton({
 	const isMounted = useIsMounted();
 	const [event, setEvent] = useState(null);
 
+	const addToCartNotAllowed =
```

Comment:
Can you do a fixup for this?

Reply:
What do you mean by fix up?

Reply:
https://git-scm.com/docs/git-commit/2.32.0#Documentation/git-commit.txt---fixupamendrewordltcommitgt